test/test_namespace/test_namespacemanager.py

```diff
@@ -484,3 +484,69 @@ def check() -> None:
     check()
     # Run a second time to check caching
     check()
+
+
+@pytest.mark.parametrize(
+    "curie, expected_uri, expected_exception, expected_exc_patt",
+    [
+        ("rdf:type", "http://www.w3.org/1999/02/22-rdf-syntax-ns#type", None, ""),
+        ("obo:IAO_0000111", "http://purl.obolibrary.org/obo/IAO_0000111", None, ""),
+        ("obo:nonexistent", "http://purl.obolibrary.org/obo/nonexistent", None, ""),
+        ("too_small", "irrelevant", ValueError, "Malformed curie argument"),
+        (
+            "blah:chair",
+            "http://blah.org/ontology#chair",
+            ValueError,
+            'Prefix "blah" not bound to any namespace',
+        ),
+        # next case only works with fix for https://github.com/RDFLib/rdflib/issues/2348
+        (":chair", "http://www.example.org/ontologies/mini-ont#chair", None, ""),
+        pytest.param(
+            # failure case that should succeed once https://github.com/RDFLib/rdflib/issues/2077 is fixed
+            "mini-ont:chair",
+            "http://www.example.org/ontologies/mini-ont#chair",
+            None,
+            "",
+            marks=pytest.mark.xfail,
+        ),
```

Comment:
This is failing for a reason unrelated to that bug though, your owl file defines: `xmlns:mini-ont="http://example.org/ontologies/mini-ont#"` - and the check here is for `http://www.example.org/` instead of `http://example.org/`.

Reply:
Fixed in https://github.com/RDFLib/rdflib/commit/598f9679b27e18755d4fbc54772f23ab0af007a7

Reply:
Ouch. Thanks for catching that!